libs/api-client-tiger/src/gd-tiger-model/VisualizationObjectModelV1.ts

```diff
@@ -33,7 +33,7 @@ export namespace VisualizationObjectModelV1 {
      */
     export interface IAttribute {
         localIdentifier: Identifier;
-        displayForm: ObjectIdentifier;
+        displayForm: AfmObjectIdentifier;
```

Comment:
can this break existing saved insights?

Reply:
It shouldn't. This just type renaming which is irrelevant for stored jsons.

Reply:
Agreed, the shape did not change, it should be safe